cluster-provision/okd/base/Dockerfile

```diff
@@ -1,24 +1,36 @@
 FROM fedora@sha256:a66c6fa97957087176fede47846e503aeffc0441050dd7d6d2ed9e2fae50ea8e
 
-# oc binary without SSL dependency
-ENV OC_PKG https://artifacts-openshift-master.svc.ci.openshift.org/repo/openshift-clients-4.0.0-0.alpha.0.1969.af45cda.x86_64.rpm
-
 RUN dnf install -y \
-${OC_PKG} \
 libvirt \
 libvirt-devel \
 libvirt-daemon-kvm \
 libvirt-client \
 qemu-kvm \
 openssh-clients \
 haproxy \
+jq \
 virt-install \
 socat \
 selinux-policy \
 selinux-policy-targeted \
-httpd-tools && \
+httpd-tools \
+python2-pip && \
 dnf clean all
 
+RUN pip2 install yq
+
+# install golang
+ENV GO_VERSION 1.12.12
+RUN curl https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz -o go.tar.gz && \
```

Comment:
This is going to make okd- providers bigger, but not much though, is that ok ?

Reply:
the difference is 300MB, IMHO it is not critical when we have images of 14Gb :(